tests/run-purecap.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+ARCH=$1
+
+if [ "$ARCH" = "riscv64" ] || [ "$ARCH" = "morello" ];; then
+   NETDEV="user,id=net0,smb=$HOME/cheri<<<source_root@ro:$HOME/cheri/build<<<build_root:$HOME/cheri/output<<<output_root@ro:$HOME/cheri/output/rootfs-$ARCH-purecap<<<rootfs,hostfwd=tcp::10021-:22"
+   $HOME/cheri/output/sdk/bin/qemu-system-"$ARCH"cheri -M virt -m 2048 -nographic -bios bbl-"$ARCH"cheri-virt-fw_jump.bin -kernel $HOME/cheri/output/rootfs-$ARCH-purecap/boot/kernel/kernel -drive if=none,file=$HOME/cheri/output/cheribsd-$ARCH-purecap.img,id=drv,format=raw -device virtio-blk-device,drive=drv -device virtio-net-device,netdev=net0 -netdev $NETDEV -device virtio-rng-pci
```

Comment:
With buildbot, one thing we have to be careful about is that, IIRC, files dumped in the buildbot user's home directory won't be flushed between runs. In other words, this can create some weird caching effects. It might be worth asking @vext01 about this, as I might be wrong.

Reply:
What if we decide to leave the `worker` with a 1-week old SDK and imgs? And I can add some scheduled tasks to refresh it when needed. In this way, we can always have buildbot running and prevent any future issues.

Reply:
> With buildbot, one thing we have to be careful about is that, IIRC, files dumped in the buildbot user's home directory won't be flushed between runs. In other words, this can create some weird caching effects. It might be worth asking @vext01 about this, as I might be wrong.

Right. So the SDK must be put elsewhere.

Reply:
At least at first, we want to avoid all caching: it'll bite us in weird ways. Later, once we understand what's going on, we can try and be a bit more clever and cache things.

Reply:
> Right. So the SDK must be put elsewhere.

`.buildbot.sh` should download the SDK, build it etc etc. I know it'll be slow, but that doesn't matter for now.

Reply:
> > Right. So the SDK must be put elsewhere.
> 
> `.buildbot.sh` should download the SDK, build it etc etc. I know it'll be slow, but that doesn't matter for now.

OK. I'll put the run inside '.buildbot.sh'. I need to identify an extra target or maybe use the same `nohup` trick with `cheribuild`.